custom-prom.md

```diff
@@ -55,14 +59,6 @@ To confirm this job is successfully scraped by Prometheus, you can view the Targ
 
 <a name="recording-rules"></a>
 
-### Recording Rules
```

Comment:
I think we're ready here, given it's been nearly 5 months

Reply:
We're ready!